### ISSUE TITLE
command 'synthwave84.enableNeon' not found

### ISSUE THREAD

**Issue by frizbee** [robb0wen/synthwave-vscode#147]
getting this error after `cmd+shift+P` and click on `Enable Neon Dreams in right bottom corner.

Version: 1.37.1
Commit: f06011ac164ae4dc8e753a3fe7f9549844d15e35
Date: 2019-08-15T16:16:34.800Z
Electron: 4.2.7
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Darwin x64 19.3.0
MacOS 10.15.3

**Comment by fraincs** [comment]
Getting the same error.

**Comment by MichalStachnik** [comment]
Same here.

**Comment by rzmz** [comment]
After upgrade to VS Code Version: 1.56.0 I get the same error, it was functioning previously.

**Comment by carl-boisvert** [comment]
I still have that error :/

**Comment by marcosrausch** [comment]
Same...

**Comment by iHeadway** [comment]
There is a solution to get write permissions to the VSC folder and make a clean install (delete extension and folder) after but it
didn't work for me.

**Comment by dyvue** [comment]
You need to downgrade your VS Code version below 1.60, maybe 1.59 is the best option

**Comment by iHeadway** [comment]
> You need to downgrade your VS Code version below 1.60, maybe 1.59 is the best option

Yes, already downgraded my vsc to 1.59 There is a newer topic about this error https://github.com/robb0wen/synthwave-vscode/issues/222 the problem is that the Windows version will break soon too.

**Comment by negative0101** [comment]
Same problem here. Can we get an update?
